content/news.pml

```diff
@@ -39,6 +39,63 @@
 
 <!----------------------------------------------------------->
 
+<div class="news-item">
+  28 July 2021: Proof that seL4 enforces integrity established for RISC-V
+</div>
+<div>
+ <p>
+   <img src="/Foundation/Membership/LOGOS/RISC-V.svg"
+   style="width: 20%;  padding-left:10px;  float:right"
+	alt="RISC-V logo">
+
+   The assurance story for <a href="https://sel4.systems">seL4</a>
+   on <a href="https://riscv.org">RISC-V</a> keeps building. We first
```

Comment:
Maybe re-word this to "First, we formally proved .."

Reply:
This is @lsf37's/@june-andronick's text, I'll leave this to them

Reply:
I don't think we should do this one, it changes the emphasis so much that it changes the meaning (it will then sound like we did all of this now instead of earlier)

Reply:
Agreed